src/Project.php

```diff
@@ -133,6 +133,16 @@ public function releaseBuilds()
     }
 
     /**
+     * Get release builds of this project
+     *
+     * @return ReleaseBuildInfo
```

Comment:
Not sure this is quite right? It looks like it is returning the raw data from the client, which probably won't be an instance of ReleaseBuildInfo?